ladybug/sunpath.py

```diff
@@ -736,38 +940,37 @@ class Sun(object):
         * data
         * is_during_day
         * sun_vector
+        * sun_vector_reversed
```

Comment:
Why not just leave this to user to reverse the sun vector if needed?

Reply:
There are 5-6 places where I need the reversed sun vector to generate different parts of the sunpath geometry or to generate points that get plotted on this geometry. So it definitely seemed worthwhile to store the reversed vector on the `Sun` object rather than regenerating it each time. If you think this may confuse people, I can store it just as a hidden property and use it internally within the sunpath module.

I think I'll merge the PR for now but, if you think it's helpful to keep the property hidden, just let me know here and I'll send another PR that hides it.

Reply:
> If you think this may confuse people, I can store it just as a hidden property and use it internally within the sunpath module.

I think it will. Having it removed or hidden will be helpful.